api-gateway/kong/kong.conf

```diff
@@ -0,0 +1,13 @@
+# Open listening ports
+proxy_listen = 0.0.0.0:8000, 0.0.0.0:8443 ssl
```

Comment:
Are we keeping the port 8000?

Reply:
It is configured in the examples by environment variable when you are going to use http or https in some domain, so I let localhost be the simplest and because of the examples also for localhost that always use 8000